sympy/sets/tests/test_sets.py

```diff
@@ -1123,8 +1156,11 @@ def test_issue_10113():
 
 
 def test_issue_10248():
-    assert list(Intersection(S.Reals, FiniteSet(x))) == [
-        (-oo < x) & (x < oo)]
```

Comment:
This result is wrong because it should not give the conditional that an element can exist in another set.

Reply:
#16362